release/release_gold.sh

```diff
@@ -97,6 +98,255 @@ function main {
 	#lc_time_run upload_to_docker_hub
 
 	lc_time_run add_patcher_project_version
+
+	lc_time_run reference_new_releases
+}
+
+function reference_new_releases {
+	prepare_branch_to_commit_from_master "${_PROJECTS_DIR}/liferay-jenkins-ee/commands" "new_releases_branch"
+
+	if [ "${?}" -ne 0 ]
+	then
+		lc_log ERROR "Unable to prepare the next release references branch."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_BAD}"
+	fi
+
+	if [[ "${_PRODUCT_VERSION}" != *q* ]]
+	then
+		lc_log INFO "Skipping the update ont he references on the liferay-jenkings-ee repository."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	fi
+
+	if [[ "${_PRODUCT_VERSION}" == release-* ]]
+	then
+		local previous_quarterly_release_branch_name="$(grep "extraapps.app.server.versions" "build.properties" | tail -1 | sed "s/.*\[\(.*\)\].*/\1/")"
+
+		sed \
+			-i "s/^		${previous_quarterly_release_branch_name}$/		${previous_quarterly_release_branch_name},\\\\/" \
+			"build.properties"
+
+		sed \
+			-i "/${previous_quarterly_release_branch_name},/a\		${_PRODUCT_VERSION}" \
```

Comment:
@kiwm, can you please check why this `/a` formatting is different from the others?